README.md

```diff
@@ -1 +1,40 @@
-# best-by-landing
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Please edit README:
- add the project's name
- add instructions how to run the project (npm ci, npm run dev), removing everything except npm
- remove tips for Next.js project which were generated automatically